docs/api.md

```diff
@@ -552,6 +552,79 @@ Output content of the table to a `pandas.DataFrame`.
     
     Maximum number of rows to return. Defaults to -1, i.e. all.
 
+## API Client
+
+
+### `sematic.client.block_on_run`
+
+
+Block on the run with the given id until it is in a terminal state.
+
+Terminal states include successful completion, failure and cancellation.
+Only successful completion returns without error, other terminal states
+result in `RuntimeError` being raised.
+
+#### Parameters
+
+- `run_id`: str
+
+    The id of the run to block on.
+
+- `delay_seconds`: float
```

Comment:
Please rename this to `polling_interval`.
`delay` suggests a one-time delay at the beginning.

Reply:
done (`polling_interval_seconds` to keep the units unambiguous).